### PR TITLE
[docs]: implement recommended community standards

### LINKED ISSUE #10: Implement recommended community standards to the project
Hello @colbyfayock 

I was going though this project and noticed that there are some missing items based on the [Github Community Standards recommendations](https://github.com/colbyfayock/next-cloudinary/community) that would make the project contributor friendly, e.g. a Licence, an issue template and a pull request template.

Let me know if this is something you would like to implement.

### PR DESCRIPTION
**Pull request by orama254**
Adds the recommended GitHub community standards to make the project contributor friendly.

fixes #10 

You can review these suggestions and maybe add some preferences and I will add them to the files.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+## Type of change
+<!-- Please select all options that are applicable. -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+---
+# Checklist:
```
Comment (colbyfayock) [inline .github/pull_request_template.md:17]: can we add do this checklist Tests? there's not a testing suite yet but that's being added in another active Issue

**Comment by colbyfayock** [comment]
thanks @orama254 just a quick request!

**Comment by colbyfayock** [comment]
also once we're good feel free to implement this over at https://github.com/colbyfayock/netlify-plugin-cloudinary/ if you'd like

File: .github/pull_request_template.md (hunk 1 above)
Comment (orama254) [reply .github/pull_request_template.md:17]: @colbyfayock 
To understand correctly, you are requesting that we change the current title from "checklist" to "do this checklist tests"

OR 

Add a checklist item where one is supposed to run a specific test (which is an active issue being worked on)

Comment (colbyfayock) [reply .github/pull_request_template.md:17]: #2 - add a checklist item about testing, i was thinking more about ADDING tests, but running tests make sense too

Comment (orama254) [reply .github/pull_request_template.md:17]: @colbyfayock I've made the changes as requested, feel free to review the latest commit

**Comment by colbyfayock** [comment]
looks great, thanks @orama254

**Comment by colbyfayock** [comment]
@all-contributors please add @orama254 for tool

**Comment by orama254** [comment]
Thanks @colbyfayock I'll be sending an email as requested.

Happy Hacktoberfest🎃

**Pull request merged**